src/test/java/cz/czechitas/AdminPage.java

```diff
@@ -0,0 +1,29 @@
+package cz.czechitas;
+
+import org.openqa.selenium.*;
+
+public class AdminPage {
+
+    WebDriver driver;
+
+    public AdminPage(WebDriver driver) {
+        this.driver = driver;
+
+    }
+
+    public void openPage() throws InterruptedException {
+        driver.navigate().to("http://czechitas-shopizer.azurewebsites.net/admin/logon.html");
+        Thread.sleep(5000L);
+
+    }
+
+    public void login(String text, String text2) {
+        WebElement name = driver.findElement(By.name("username"));
```

Comment:
neni to nezbytne nutne, ale casto se ty By selectory vytahuji do jako promenna na uroven tridy. Ten kod je potom totiz o neco malo prehlednejsi.